.github/workflows/ci_pr.yml

```diff
@@ -8,92 +8,23 @@ on:
   workflow_dispatch:
 
 jobs:
-  test-python-2_6-and-3_4-versions:
-
-    strategy:
-      fail-fast: false
-      matrix:
-        include:
-          - python-version: "2.6"
-          - python-version: "3.4"
-
+  execute-tests:
     name: "Python ${{ matrix.python-version }} Unit Tests"
     runs-on: ubuntu-20.04
-    container:
-      image: ubuntu:24.04
```

Comment:
The python-setup action breaks when the VM is Ubuntu > 20. At some point, we will need to look into that.